tests/test_packaging.py

```diff
@@ -16,8 +16,18 @@ def test_requirements_files():
             f'{line.name}{"".join(line.specs[0])}' for line in rp.parse(req)
         }
 
-    with open(file_relative_path(__file__, "../requirements-dev-base.txt")) as req:
-        requirements_dev_base = {
+    with open(file_relative_path(__file__, "../requirements-dev-lite.txt")) as req:
+        requirements_dev_lite = {
+            f'{line.name}{"".join(line.specs[0])}' for line in rp.parse(req)
+        }
+
+    with open(file_relative_path(__file__, "../requirements-dev-contrib.txt")) as req:
+        requirements_dev_contrib = {
+            f'{line.name}{"".join(line.specs[0])}' for line in rp.parse(req)
+        }
```

Comment:
Same note here - the granularity we're adding adds a lot of verbosity and I'd love to simply things into a loop/function/dict if possible.

Reply:
Not changing this file

Reply:
I ended up refactoring it...